app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns.scss

```diff
@@ -1,4 +1,4 @@
-$dropdown-height: 373px; // Allows for entire hero_simple text to display from $screen-md and up
+$dropdown-height: 345px; // Allows for entire hero_simple text to display from $screen-md and up
```

Comment:
Wait, my bad you can disregard the above... I hadn't commented out the storage bar to view it.